platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml

```diff
@@ -29,7 +29,7 @@
   loop: "{{ peers }}"
   loop_control:
     loop_var: peer
-  when: peer.cli == "enabled"
+  when: peer.cli is defined and peer.cli == "enabled"
```

Comment:
Why do we have 'is defined' check?
The tag will be there in all the peers for sure and it will either be enabled or disabled.

Reply:
This is for backward compatibility. cli tag is not mandatory

Reply:
If some network configurations files do not have cli tags, so it will not fail. As peer cli is an additional functionality option

Reply:
Okay